sdk/core/Azure.Core/src/RequestContent.cs

```diff
@@ -175,7 +175,10 @@ private sealed class StreamContent : RequestContent
             public StreamContent(Stream stream)
             {
                 if (!stream.CanSeek)
+                {
                     throw new ArgumentException("stream must be seekable", nameof(stream));
```

Comment:
```suggestion
                    throw new ArgumentException("Stream must be seekable.", nameof(stream));
```

Reply:
Though, we may find this undesirable because it's an existing error... but the lack of proper sentence makes me sad.